flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -387,6 +387,7 @@ function check_logs_for_exceptions {
    | grep -v  "WARN  org.apache.flink.shaded.akka.org.jboss.netty.channel.DefaultChannelPipeline" \
    | grep -v 'INFO.*AWSErrorCode' \
    | grep -v "RejectedExecutionException" \
+   | grep -v "CancellationException" \
```

Comment:
Why has this had to be added? Is it caused by one of your change?

Reply:
My change causes `CancellationException` to be logged at `INFO` level as any other exception (which I think is fine).

In general, I think `CancellationException` in logs shouldn't fail the tests (unless it's cause by some other exception, which will be detected by script). So I made it a separate commit.

Reply:
In that case can you squash this with the commit that introduced the problem?

Reply:
Done.